### ISSUE TITLE
Fixed `npm install` in docker, updated insecure packages !6 - [merged]

### ISSUE THREAD

**Issue by pxpeterxu** [coursetable/coursetable#39]
In GitLab by @hsource on Sep 14, 2019, 01:26

_Merges harry/docker-npm-audit -> master_

`npm install` by default implicitly checks whether any packages have security vulnerabilities, and fails to install if that's the case. This change:

1. Fixes the security vulnerabilities
2. Makes `npm install` run via the `npm run install-and-watch` continue even if there are security vulnerabilities

For reference: this is the problem we'd run into before this change.

```
/var/www/html # npm install
npm WARN acorn-dynamic-import@4.0.0 requires a peer of acorn@^6.0.0 but none is installed. You must install peer dependencies yourself.
npm WARN ajv-keywords@3.4.0 requires a peer of ajv@^6.9.1 but none is installed. You must install peer dependencies yourself.
npm WARN babel-eslint@10.0.1 requires a peer of eslint@>= 4.12.1 but none is installed. You must install peer dependencies yourself.

audited 40049 packages in 25.132s
found 1209 vulnerabilities (1 moderate, 1208 high)
  run `npm audit fix` to fix them, or `npm audit` for details
```

**Comment by pxpeterxu** [comment]
In GitLab by @hsource on Sep 14, 2019, 01:26

changed title from **Fixed `npm install` in docker, updated insecure packages** to **Fixed `npm install` in docker, updated insecure packages{+ !6+}**

**Comment by pxpeterxu** [comment]
In GitLab by @hsource on Sep 14, 2019, 01:26

changed the description

**Comment by pxpeterxu** [comment]
In GitLab by @hsource on Sep 14, 2019, 01:26

merged

**Comment by pxpeterxu** [comment]
In GitLab by @hsource on Sep 14, 2019, 01:26

mentioned in commit bb4e9f8b35a2c66064d9da9191e49f7ffdb92db3

**Comment by hsheth2** [comment]
Merged in bb4e9f8b35a2c66064d9da9191e49f7ffdb92db3.
